src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -0,0 +1,184 @@
+import _ from "lodash"
```

Comment:
__[jshint]__
*  'import' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->